app/code/Magento/SalesRule/Test/Mftf/Test/EndToEndB2CGuestUserTest.xml

```diff
@@ -38,6 +38,45 @@
             <argument name="total" value="447.00"/>
         </actionGroup>
 
+        <actionGroup ref="StorefrontCancelCouponActionGroup" stepKey="couponCancelCoupon" after="couponCheckCartWithDiscount"/>
```

Comment:
Please, revert changes, files in the SalesRule module are part of Magento core, not MSI project

Reply:
Reverted changes